react/game/src/app/screens/About/styles.scss

```diff
@@ -0,0 +1,36 @@
+
+
+.title {
+  font-size: 25px;
+  font-weight: bold;
+  display: block;
+  margin-bottom: 30px;
+  text-align: center;
+  text-transform: uppercase;
+  
+}
+
+.about {
+  display:flex;
+  flex-wrap: wrap;
+}
+
+.paragraph {
+  align-items: center;
+  display: flex;
+  line-height: 1.2;
+  margin: 10px;
+ 
+  
+
+
+  span {
```

Comment:
This span affects to all `span` of the entire app and we don't want that. U need to define a class style for the span of this screen component